portsyncd/portsyncd.cpp

```diff
@@ -154,12 +154,17 @@ void handlePortConfigFile(ProducerTable &p, string file)
         }
 
         istringstream iss(line);
-        string alias, lanes;
-        iss >> alias >> lanes;
+        string name, lanes, alias;
+        iss >> name >> lanes >> alias;
```

Comment:
If there is no alias on one line, can we assume that the whole stream has no alias?